docs/conf/writing.md

```diff
@@ -55,20 +55,28 @@ picked up by the serve tasks of `@backstage/cli` for local development, and are
 injected by the entrypoint of the nginx container serving the frontend in a
 production build.
 
-## File Resolution
+## Configuration Files
 
 It is possible to have multiple configuration files, both to support different
 environments, but also to define configuration that is local to specific
-packages.
+packages. The configuration files to load are selected using a `--config <path>`
+flag, and it is possible to load any number of files. Paths are relative to the
+working directory of the executed process, for example `package/backend`. This
+means that to select a config file in the repo root when running the backend,
+you would use `--config ../../my-config.yaml`.
```

Comment:
This is something that's a bit confusing btw, running `yarn start-backend --config app-config.yaml` in the repo root will refer to `packages/backend/app-config.yaml`, we could potentially resolve the config relative to the monorepo root instead of the workdir, but that's slightly more magic.